app/views/user_notification_mailer/login_notification.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="email-container">
+<div class="email-header text-2xl font-bold mb-6 text-center text-gray-700">
+  Successfully Logged In
+</div>
+<div class="email-body text-lg text-gray-600 leading-6">
+  <p class="mb-4">Hi <%= @user.first_name %>,</p>
+  <p class="mb-4">We noticed that you just logged into your account.</p>
+  <p class="mb-4">If this was you, you can safely ignore this email. If you did not log in, please contact our support team immediately.</p>
+</div>
+<div class="email-footer text-sm text-gray-500 text-center mt-6">
+  <p>Thank you,</p>
+  <p>Your Company Team</p>
+</div>
+</div>
```

Comment:
add eol to all the files where it doesn't exits at the end of the code

Reply:
bhai the EOL is already there in every file, 
I dont know why github shows an waring